instrumentation/opentelemetry_broadway/README.md

```diff
@@ -18,6 +20,39 @@ def start(_type, _args) do
 end
 ```
 
+### With Trace Propagation
```

Comment:
Are there backends that don't support metadata? If possible we should list out which ones it doesn't work with and let them know they'll have to package up the context into their messages in that case. Though maybe all of them do support metadata? That'd be nice.